test/test.py

```diff
@@ -152,6 +152,53 @@ def test_dl_transpose4(self):
         self.assertEqual(self._dl('bca', 'ab'), 2.5)
 
 
+class TestClev_noinit(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def _lev(self, x, y):
+        return lev(x, y)
+
+    def _osa(self, x, y):
+        return osa(x, y)
+
+    def _dl(self, x, y):
+        return dam_lev(x, y)
+
+    def test_lev_noinit(self):
+        self.assertEqual(self._lev('1234', '1234'), 0.0)
+        self.assertEqual(self._lev('', '1234'), 4.0)
+        self.assertEqual(self._lev('1234', ''), 4.0)
+        self.assertEqual(self._lev('', ''), 0.0)
+        self.assertEqual(self._lev('1234', '12'), 2.0)
+        self.assertEqual(self._lev('1234', '14'), 2.0)
+        self.assertEqual(self._lev('1111', '1'), 3.0)
+
+    def test_osa_noinit(self):
+        self.assertEqual(self._osa('1234', '1234'), 0.0)
+        self.assertEqual(self._osa('', '1234'), 4.0)
+        self.assertEqual(self._osa('1234', ''), 4.0)
+        self.assertEqual(self._osa('', ''), 0.0)
+        self.assertEqual(self._osa('1234', '12'), 2.0)
+        self.assertEqual(self._osa('1234', '14'), 2.0)
+        self.assertEqual(self._osa('1111', '1'), 3.0)
+
+    def test_dl_noinit(self):
+        self.assertEqual(self._dl('', ''), 0)
+        self.assertEqual(self._dl('', 'a'), 1)
+        self.assertEqual(self._dl('a', ''), 1)
+        self.assertEqual(self._dl('a', 'b'), 1)
+        self.assertEqual(self._dl('a', 'ab'), 1)
+        self.assertEqual(self._dl('ab', 'ba'), 1)
+        self.assertEqual(self._dl('ab', 'bca'), 2)
+        self.assertEqual(self._dl('bca', 'ab'), 2)
+        self.assertEqual(self._dl('ab', 'bdca'), 3)
+        self.assertEqual(self._dl('bdca', 'ab'), 3)
+
+
 if __name__ == '__main__':
```

Comment:
Since `python setup.py test` and `tox` work now, I wonder if it makes sense to just remove this logic entirely.